ZenPacks/zenoss/OpenStackInfrastructure/modeler/plugins/zenoss/OpenStackInfrastructure.py

```diff
@@ -455,9 +457,15 @@ def process(self, device, results, unused):
 
         # Find all hosts which have a neutron agent on them.
         for agent in results['agents']:
-            host_id = prepId("host-{0}".format(agent['host']))
+            log.debug("Sanitizing hostname %s", agent['host'])
```

Comment:
This is probably a little chatty.   A message that shows when the hostname was changed might be handy, but otherwise, it's not super useful